app/bundles/ConfigBundle/Translations/en_US/messages.ini

```diff
@@ -1,4 +1,16 @@
 mautic.config.header.index="Configuration"
 mautic.config.menu.index="Configuration"
 mautic.config.restricted="Set by system"
-mautic.config.notwritable="The configuration file is not writable! Changes will not be saved."
\ No newline at end of file
+mautic.config.notwritable="The configuration file is not writable! Changes will not be saved."
+mautic.sysinfo.header.index="System Info"
+mautic.sysinfo.menu.index="System Info"
+mautic.sysinfo.tab.phpinfo="PHP Info"
+mautic.sysinfo.tab.folders="Folder Permissions"
+mautic.sysinfo.folders.title="Following folders must be writable for Mautic in order to work correctly"
```

Comment:
Try this instead:

`The following folders must be writable for Mautic to work correctly`